hack/errata.py

```diff
@@ -83,13 +91,75 @@ def notify(message, webhook=None):
         print(message)
         return
 
+    msg_text = '<!subteam^STE7S7ZU2>: {fulladvisory} shipped {when}: {synopsis}'.format(**message)
+    if  message['approved_pr']:
+        msg_text += "\nPR {approved_pr} has been approved".format(**message)
+
     urllib.request.urlopen(webhook, data=urllib.parse.urlencode({
         'payload': {
-            'text': '<!subteam^STE7S7ZU2>: {fulladvisory} shipped {when}: {synopsis}'.format(**message),
+            'text': msg_text,
         },
     }).encode('utf-8'))
 
 
+def get_open_prs_to_fast(repo):
+    query_params = {
+        'state': 'open',
+        'base': 'master',
+        'author': 'openshift-bot',
+        'sort': 'created',
+    }
+    for pr in repo.get_pulls(**query_params):
+        try:
+            # Skip unknown PRs
+            if not pr.title.startswith("Enable "):
+                continue
+            # Ignore PRs which don't target fast
+            if pr.title.split(" ")[3] != "fast":
+                continue
+            yield pr
+        except Exception as e:
+            _LOGGER.warn("Failed to parse {}: {}".format(pr.number, e))
+
+
+def extract_errata_number_from_body(body):
+    ERRATA_MARKER = 'https://errata.devel.redhat.com/advisory/'
+    first_line = body.split('\n')[0]
+    links = [
+        x for x in first_line.split(' ') if x.startswith(ERRATA_MARKER)
+    ]
+    if len(links) == 0:
+        _LOGGER.warn("No links found in PR body: {}".format(body))
+        return None
+    errata_num = links[0].rsplit('/', 1)[-1]
+
+    try:
+        return int(errata_num)
+    except ValueError:
+        _LOGGER.warn("Failed to convert PR number to int: {}".format(errata_num))
+        return None
+
+
+def lgtm_fast_pr_for_errata(githubrepo, githubtoken, message):
+    if not githubtoken:
+        _LOGGER.debug("Skipping fast PR check: no github token set")
+        return
+
+    github_object = github.Github(githubtoken)
+    repo = github_object.get_repo(githubrepo)
+
+    for pr in get_open_prs_to_fast(repo):
+        errata_num = extract_errata_number_from_body(pr.body)
+        if not errata_num or errata_num != message.get('errata_id'):
+            continue
+
+        _LOGGER.debug("Found PR #{} promoting to fast for {}".format(pr.number, errata_num))
+        msg = "Autoapproving PR to fast after the errata has shipped\n/lgtm"
```

Comment:
@LalatenduMohanty points out that we need `/hold cancel` here too (because the candidate promotion has presumably merged long before).  Or we can teach the ART tooling to stop setting holds on the fast PRs, if we want a way to block a PR from auto-merging for whatever reason.